aws/components/apigateway/setup.ftl

```diff
@@ -303,21 +303,57 @@
             outputs={}
             dependencies=apiId
         /]
+
+        [#-- Throttling Configuration --]
+        [#local openapiIntegrations = 
+            getOccurrenceSettingValue(occurrence, [["apigw"], ["Integrations"]], true)]
+
+        [#local methodSettings = [
+            {
+                "HttpMethod": "*",
+                "ResourcePath": "/*",
+                "LoggingLevel": "INFO",
```

Comment:
Don't want this I think unless explicitly defined - unless mandatory

Reply:
Amended with a wrapper if-statement, and instantiating an empty array.